docs/index.rst

```diff
@@ -136,6 +136,36 @@ Sentry, all you have to do is add a requirement on ``IRavenClient`` in your clas
             this.ravenClient = ravenClient;
         }
     }
+    
+Breadcrumbs
+-----------
+
+Sentry supports a concept called Breadcrumbs, which is a trail of events which happened prior to an issue. Often times these events are very similar to traditional logs, but also have the ability to record more rich structured data. `More here <https://docs.sentry.io/learn/breadcrumbs/>`
```

Comment:
Can you perhaps link the word `Breadcrumbs` in the first sentence to the docs instead of `More here`?